web/libs/editor/src/utils/billing.ts

```diff
@@ -0,0 +1,3 @@
+import { isFF, FF_SELF_SERVE } from "./feature-flags";
```

Comment:
this may be better positioned in the `core` library undersomething like `web/libs/core/src/lib/utils/billing.ts`
this way it can be reused in LSO and LSE with out redefining the function in both as i see the definition in LSE already exists but not in LSO

Reply:
this is a minor nitpick - i wouldnt hold up the PR over this

Reply:
I'll attempt this refactor in a follow up! Agree it would be better but I verified the code pretty carefully as is, so would rather push this to a future diff :)